code/modules/antagonists/morph/morph.dm

```diff
@@ -81,7 +81,7 @@
 /mob/living/simple_animal/hostile/morph/ClickOn(atom/A)
 	if(throwatom)
 		RemoveContents(throwatom, TRUE)
-		throwatom.safe_throw_at(A, throwatom.throw_range, throwatom.throw_speed, src, null, null, null, move_force)
+		throwatom.safe_throw_at(A, throwatom.throw_range, throwatom.throw_speed, src, null, null, null, ismob(throwatom) ? MOVE_FORCE_VERY_WEAK : move_force)
```

Comment:
I kind of thing that the throw range should be reduced instead, if anything. Morphs are not insanely strong as an antagonist and having a relatively short stun isn't too bad, especially considering that the bodies can be caught. The morph is about using random objects as weapons since they are very weak and easy to counter alone.

Ingesting a burning mob should set the morph on fire, or extinguish it (since morphs can't be set on fire).

I agree that morphs hoarding bodies is cringe, but will this stop them from doing that? Don't they get the ability to consume bodies anyway for health making this a choice between either a weapon or health.

Reply:
Consuming was changed from dusting to husking, so now they can drop the body. However having it be a free stun incentivizes keeping it.

Reply:
> Ingesting a burning mob should set the morph on fire, or extinguish it (since morphs can't be set on fire).

Well it does, ish. Problem being the bodies had a virus that made them burn forever. It's a cool use case that one time but the throwing was just ridiculously overpowered